docs/developer/products.mdx

```diff
@@ -3,43 +3,83 @@ title: Querying Product Information
 sidebar_label: Products
 ---
 
-import GraphQL from "@site/components/GraphQL";
-
 ## Introduction
 
 This guide describes how to obtain products from the Saleor GraphQL API.
 
 You can either retrieve a single product or a list of products. You may require a list of products in many situations, for example, when you need to simply display the catalog in your storefront, or to provide a third party service with a list of products available in your store.
 
+
+## Multichannel and products
+
+Customers can only fetch products from available channels. Because listing channels is only available for staff users, you will need to provide channel slugs to you storefront.
```

Comment:
I'm not sure that's true. Did you check it?
In some point of developing 3.0 we assume that some staff user should be able to display all product to assign to Draft order or Voucher/Sale.